lib/list.gd

```diff
@@ -1002,6 +1002,34 @@ DeclareOperation( "PositionsProperty", [ IsList, IsFunction ] );
 DeclareOperation( "PositionBound", [ IsList ] );
 
 
+#############################################################################
+##
+#O  PositionsBound( <list> ) . . . . . . . . . positions of all bound entries
+##
+##  <#GAPDoc Label="PositionsBound">
+##  <ManSection>
+##  <Oper Name="PositionsBound" Arg='list'/>
+##
+##  <Description>
+##  returns the set of all indices for which an element is bound in the list
+##  <A>list</A>.
+##  <P/>
+##  <Example><![CDATA[
+##  gap> PositionsBound([1,2,3]);
+##  [ 1, 2, 3 ]
+##  gap> PositionsBound([,1,,3]);
+##  [ 2, 4 ]
+##  gap> PositionsBound([]);
+##  []
+##  ]]></Example>
+##  </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+##
+DeclareOperation( "PositionsBound", [ IsList ] );
```

Comment:
As Frank pointed out, this should be turned into a global function for optimal performance.

Reply:
In that case it should not refer back to PositionsProperty (since that is not a global function).